examples/matmul.mojo

```diff
@@ -162,8 +160,7 @@ fn matmul_tiled_parallelized(C: Matrix, A: Matrix, B: Matrix):
                     C.store[nelts](
                         m,
                         n + x,
-                        C.load[nelts](m, n + x)
-                        + A[m, k] * B.load[nelts](k, n + x),
+                        C.load[nelts](m, n + x) + A[m, k] * B.load[nelts](k, n + x),
```

Comment:
Jack - this line was formatted by `mojo format`; otherwise the code would fail the lint step.

Reply:
@dualvtable I did a `mojo format` on all the files and that's what it tuned into, so decided to leave it in the PR. This repo doesn't have those lint checks on it.